data/txHandler.go

```diff
@@ -90,19 +94,58 @@ func MakeTxHandler(txPool *pools.TransactionPool, ledger *Ledger, net network.Go
 		backlogQueue:          make(chan *txBacklogMsg, txBacklogSize),
 		postVerificationQueue: make(chan *txBacklogMsg, txBacklogSize),
 		net:                   net,
+		streamVerifierChan:    make(chan verify.UnverifiedElement),
+		streamReturnChan:      make(chan verify.VerificationResult),
```

Comment:
The pipeline has buffers, that is why we do not need extra buffers here. 
Buffers need to be accompanied by dropping (in case of overflow) and reporting of the dropped. So it is not as simple as just making the chan wider. 
The buffers are: `backlogQueue` and `postVerificationQueue`.
It is important that you are convinced that if `streamVerifierChan` or `streamReturnChan` are blocked, `backlogQueue` and `postVerificationQueue` will absorb the flow.

Reply:
my read of streamVerifierChan and streamReturnChan is that it shouldn't be a chan but just a function call with return. all the `StreamVerifier.sendResult` happen immediately in the flow of processing one posted verification request.

Reply:
I am open to consider alternatives to the currently implemented input/output channels. 
Please post here with pros/cons of the current channels and the alternative approaches.

Reply:
re-reading the code I still think StreamVerifier can be a reasonable chunk of state to have in an object, but data should be passed to it by just calling a function instead of message passing on a chan, and data coming back from it would be better if it didn't have to bounce through a go thread unpacking the result from one chan and putting it on a different chan (`processTxnStreamVerifiedResults()`)

Reply:
Discussed this with Brian. We agreed to keep the message passing to the StreamVerifer as is, but to get rid of the extra channel redirection on the return path.
I will update the code.